Makefile

```diff
@@ -1,4 +1,15 @@
 export IPFS_API ?= v04x.ipfs.io
 
-deps:
-	env GO111MODULE=on go mod download
+gx:
+	go get -u github.com/whyrusleeping/gx
+	go get -u github.com/whyrusleeping/gx-go
+
+deps: gx
+	gx --verbose install --global
+	gx-go rewrite
+
+publish:
+	gx-go rewrite --undo
+
+mod_deps:
+	env GO111MODULE=on go get ./...
```

Comment:
I don't think this does what you think it does.

Reply:
@anacrolix then what should it be? I can't see any updates to how the Makefile works on #251?

Reply:
You probably want `go build ./...`, `go mod download`, or nothing at all.

Reply:
Also not a blocker.